domainobjs/ts/publicKey.ts

```diff
@@ -0,0 +1,155 @@
+import * as assert from "assert";
+import {
+    SNARK_FIELD_SIZE,
+    hashLeftRight,
+    packPubKey,
+    unpackPubKey,
+    PubKey as RawPubKey,
+} from "maci-crypto";
+import { SERIALIZED_PUB_KEY_PREFIX } from "./constants";
+
+/**
+ * @notice A class representing a public key
+ * @note This is a MACI public key, which is not to be
+ * confused with an EVM public key.
```

Comment:
What do you mean by that EVM public key? Technically EVM public key is not an established term

Reply:
What would you use instead? this is your Ethereum public key, which works on any EVM chains

Reply:
I'd say "Ethereum public key" might be best

Reply:
fyi: since there is an ECDSA public key using the secp256k1 curve, the Ethereum address of an EOA is generated by taking a keccak hash in 64-character hex format of the public key and dropping the first 24 characters. Note that the EVM doesn't directly store the public key, although they can be derived from the r, s, v fields in a transaction (https://ethereum.stackexchange.com/questions/13778/get-public-key-of-any-ethereum-account)